### ISSUE TITLE
missing tools

### ISSUE THREAD

**Issue by bgruening** [bio-tools/content#479]
The following is a list of tools that are missing or are not easy to map. A few of them are high-profile tools.

xref: https://github.com/galaxyproject/tools-iuc/pull/3411/files

annotatemyids
anndata
barcode_splitter
basil
bax2bam
bctools
berokka
python-bioext
biom_format
bwameth
cat
chromeister
codeml
colibread
crispr_studio
crossmap
datamash
ena-upload-cli
enasearch
export2graphlan
fargene
fastani
fastp
fastqe
fermi2
fitlong
flair
mutect2
gecko
genehunter
genrich
goslimmer
gprofiler2
graphlan
gubbins
hansel
hicexplorer
hivclustering
humann2
idr
iqtree
ivar
jq
kcalign
kma
lofreq
lorikeet
lumpy_sv
maxbin2
medaka
dreme -- meme_meme and meme_fimo exist
meme_chip
miniasm
mitobim
moabs
mothur
msaboot
multigps
nanoplot
nanopolishcomp
nonpareil
novoplasty
nudup
ococo
odgi
picrust
pipelign
pizzly
porechop
progressivemauve
proteinortho
pureclip
pyGenomeTracks
qfilt
raven-assembler
sarscov2formatter
sarscov2summary
seacr
seqwish
shasta
slamdunk
alleyoop
snp-dists
socru
spaln
spyboat
stacks2  --> is that stacks1
star fusion
structureharvester
swiftlink
tb_variant_filter
tbl2gff3
tetoolkit_tetranscripts
tetyper
tn93
transtermhp
valet
vardict_java
variant_analyzer
vegan
vg
volcanoplot
vsnp
simple_weather
xpath
zerone

https://github.com/wtsi-hpag/Scaff10X

**Comment by supernord** [comment]
Hi @bgruening - is this something we could converge on and address together with the bio.tools team? For those entries that are still missing.

**Comment by supernord** [comment]
@hmenager @hansioan

**Comment by bgruening** [comment]
Of course. My fav way would be that we add those tools to this repo and bio.tools fetches those from this repo. But I'm not sure we are there yet.

**Comment by rioualen** [comment]
Hi 
Is this about mapping Galaxy tools with bio.tools entries? We could revise those, since this issue is a bit dated and we are now importing tools from the Galaxy CoDex. 
A recent related issue: [Of 1800 tools only 600 have bio.tools IDs](https://github.com/galaxyproject/galaxy_codex/issues/722)
@paulzierep
